rdkit/Chem/Subshape/SubshapeAligner.py

```diff
@@ -318,8 +313,8 @@ def GetSubshapeAlignments(self, targetMol, target, queryMol, query, builder, tgt
   def __call__(self, targetMol, target, queryMol, query, builder, tgtConf=-1, queryConf=-1,
                pruneStats=None):
     for alignment in self.GetTriangleMatches(target, query):
-      if builder.featFactory and \
-         not self._checkMatchFeatures(target.skelPts,query.skelPts,alignment):
+      if builder.featFactory and not self._checkMatchFeatures(target.skelPts, query.skelPts,
```

Comment:
another one of those examples where I think the reformatting has made the code less readable, but I won't fight for it.